Add Installation and Getting Started docs (#120)
Oneflow-Inc/libai: README.md

</details>

## Installation

> Please refer to [Installation.md](https://libai.readthedocs.io/en/latest/tutorials/Installation.html) for installation

Reviewer: ```suggestion
See [installation instructions](https://libai.readthedocs.io/en/latest/tutorials/Installation.html).
```

## Getting Started

> Please refer to [Getting Started.md](https://libai.readthedocs.io/en/latest/tutorials/Getting%20Started.html) for basic usage of LiBai

Reviewer: ```suggestion
See [Getting Started](https://libai.readthedocs.io/en/latest/tutorials/Getting%20Started.html) to learn about basic usage of LiBai.
```
